src/compute/src/sink/persist_sink.rs

```diff
@@ -85,10 +85,14 @@ where
         let persist_client = futures_executor::block_on(persist_location.open())
             .expect("cannot open persist client");
 
-        let (write, _read) = futures_executor::block_on(
-            persist_client.open::<Row, Row, Timestamp, Diff>(self.shard_id),
-        )
-        .expect("could not open persist shard");
+        let write = futures_executor::block_on(async {
+            let (write, read) = persist_client
```

Comment:
we have `open_reader` and `open_writer` variants now that only return one of the handles for exactly this reason

Reply:
:O amazing! thanks for adding those